src/components/Facility/ConsultationDetails/ConsultationFeedTab.tsx

```diff
@@ -50,7 +51,23 @@ export const ConsultationFeedTab = (props: ConsultationTabProps) => {
     }
   }, []);
 
-  const asset = preset?.asset_bed.asset_object;
+  const bedObjectId = props.consultationData.current_bed?.bed_object?.id || "";
+  const bedsQuery = useQuery(routes.listAssetBeds, {
+    query: {
+      bed_object: bedObjectId,
+      limit: 50,
+    },
+  });
```

Comment:
Use proper names to avoid confusions. You are using the Asset Beds list API and not Beds list API.

```suggestion
  const assetBedsQuery = useQuery(routes.listAssetBeds, {
    query: {
      bed_object: bedObjectId,
      limit: 50,
    },
  });
```